vestido/vestido-storefront/pages/styles.css

```diff
@@ -71,11 +71,11 @@
 }
 
 @layer base {
-  * {
+  /* * {
     @apply border-border;
-  }
+  } */
   body {
-    @apply bg-background text-foreground;
+    /* @apply bg-background text-foreground; */
     font-feature-settings: "rlig" 1, "calt" 1;
```

Comment:
Please do not commit this now, we have to solve this in another PR.

Reply:
Done